server/datastore/mysql/apple_mdm.go

```diff
@@ -770,30 +770,37 @@ func updateMDMAppleHostDB(
 ) error {
 	refetchRequested, lastEnrolledAt := mdmHostEnrollFields(mdmHost)
```

Comment:
I wonder if it would make things easier to maintain to split up this abstraction and calculate these two values separately when needed?